front/style/pages/_dashboard.scss

```diff
@@ -0,0 +1,97 @@
+.dashboard {
+    margin-bottom: 25px;
+    a {
+        text-decoration: none;
+    }
+
+    .newblue {
+        color: $blue--dark;
+    }
+
+    .button {
+        width: 199px;
+        height: 47px;
+        border: solid 1px #000000;
+        background-color: #ffffff;
```

Comment:
You should use the `$white` variable here

Reply:
done